packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -15,8 +15,10 @@ import { Icon, IIconProps } from '../../Icon';
 import { DirectionalHint } from '../../common/DirectionalHint';
 import { ContextualMenu, IContextualMenuProps } from '../../ContextualMenu';
 import { IButtonProps, IButton } from './Button.Props';
-import { IconButton } from './IconButton/IconButton';
 import { IButtonClassNames, getClassNames } from './BaseButton.classNames';
+import { getStyles as getSplitButtonStyles } from './SplitButton/SplitButton.styles';
```

Comment:
BaseButton should render the splitbutton content. But the styling for how the split looks may vary from variant to variant.